plugins/processors/enum/enum.go

```diff
@@ -37,17 +39,38 @@ func (mapper *EnumMapper) Init() error {
 	mapper.FieldFilters = make(map[string]filter.Filter)
 	mapper.TagFilters = make(map[string]filter.Filter)
 	for _, mapping := range mapper.Mappings {
+		if mapping.Field != "" && len(mapping.Fields) != 0 {
+			return fmt.Errorf("use field or fields, but not both")
+		}
 		if mapping.Field != "" {
 			fieldFilter, err := filter.NewIncludeExcludeFilter([]string{mapping.Field}, nil)
 			if err != nil {
-				return fmt.Errorf("failed to create new field filter: %w", err)
+				return fmt.Errorf("failed to create new field filter from field: %w", err)
 			}
 			mapper.FieldFilters[mapping.Field] = fieldFilter
 		}
```

Comment:
What about just adding the single `Field` to the `Fields` array?

Reply:
I do agree with @Hipska. With the deprecation this would be the best way...